internal/cmd/schema-registry/command_cluster_delete.go

```diff
@@ -40,24 +39,22 @@ func (c *command) newClusterDeleteCommand() *cobra.Command {
 }
 
 func (c *command) clusterDelete(cmd *cobra.Command, _ []string) error {
-	ctx := context.Background()
-
 	environmentId, err := c.Context.EnvironmentId()
 	if err != nil {
 		return err
 	}
 
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(ctx, environmentId)
+	clusters, err := c.V2Client.GetSchemaRegistryClustersByEnvironment(environmentId)
```

Comment:
Can an environment have more than one SR cluster now?

Reply:
cc @Anshul-Goyal

Reply:
No, we can only have one SR per environment.

Reply:
Why does this API return a list of SR clusters, then? Is that something we plan to support in the future?

Reply:
It also looks like this API is paginated too? (A bit weird, considering it will only ever return 0 or 1 results)